storagedriver/s3/s3_test.go

```diff
@@ -28,11 +30,42 @@ func init() {
 	}
 
 	s3DriverConstructor := func(region aws.Region) (storagedriver.StorageDriver, error) {
-		shouldEncrypt, err := strconv.ParseBool(encrypt)
-		if err != nil {
-			return nil, err
+		encryptBool := true
```

Comment:
`encrypt` should default to `false` like the semantic defined in the README. Also, you should remove this from the list of environment variables required to run the tests.